doc/api/net.md

```diff
@@ -112,6 +143,9 @@ The parameter `backlog` behaves the same as in
 [`server.listen(port[, hostname][, backlog][, callback])`][`server.listen(port, host, backlog, callback)`].
 
 ### server.listen(options[, callback])
+<!-- YAML
+added: v0.3.2
+-->
```

Comment:
I don’t think this overload was available back then, but with `.listen()` that’s not too easy to tell… the corresponding docs change for this one happened in 029cfc12a0736b0b03d0101d24b8a1a3023137a5, which would mean that `0.11.14` is appropiate.